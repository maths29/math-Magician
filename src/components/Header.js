import { NavLink } from 'react-router-dom';
import React from 'react';
import '../styles/header.css';

const Header = () => (
  <div>
    <div className="header">
      <h1>Math Magicians</h1>
      <div className="ulDiv">
        <ul>
          <li className="nav-links"><NavLink to="/">Home</NavLink></li>
          <li className="nav-links"><NavLink to="/calculator">Calculator</NavLink></li>
          <li className="nav-links"><NavLink to="/quote">Quotes</NavLink></li>
        </ul>
      </div>
    </div>
  </div>
);

export default Header;
