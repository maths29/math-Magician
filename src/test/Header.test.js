import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('test the top menu display', () => {
  test('check the menu snapshots', () => {
    const tree = renderer.create(
      <Router>
        <Header />
      </Router>,
    ).toJSON;

    expect(tree).toMatchSnapshot();
  });
});
