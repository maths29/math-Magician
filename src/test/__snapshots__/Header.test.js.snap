// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the top menu display check the menu snapshots 1`] = `[Function]`;
